packages/thriver-core/lib/templates/browseHappy.html

```diff
@@ -0,0 +1,16 @@
+<template name="browseHappy">
+    <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <!--[if IE 6]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 7]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 8]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!--[if IE 9]>
+                <p class="browsehappy"><span class="inner"><span class="fa">&#xf071;</span> You are using an outdated browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</span></p>
+    <![endif]-->
+    <!-- BROWSE HAPPY ENDS HERE -->
```

Comment:
This was an error. That uncommented out element should not have existed.
I've also reduced the condition to simply: `<!--[if lte IE 9]>`